src/components/GiftCard.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import withIntl from '../lib/withIntl';
+import Container from './Container';
+import { Span } from './Text';
+import Currency from './Currency';
+import Link from './Link';
+import { Flex, Box } from 'grid-styled';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import { width, height, fontSize } from 'styled-system';
+
+
+const WhiteLink = styled(Link)`
+  color: white;
+  text-decoration: underline !important;
+  &&:hover {
+    color: #eee;
+  }
+  &&:active {
+    color: #ddd;
+  }
+`;
+const Card = styled(Box)`
+  ${width};
+  ${height};
+  position: relative;
+  border-radius: 16px;
+  overflow: hidden;
+  background-image: url('/static/images/oc-gift-card-front.svg');
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-color: transparent;
+`;
+
+const ShadowCard = styled(Card)`
+  box-shadow: 0px 8px 16px rgba(20, 20, 20, 0.12);
+`;
+
+const Text = styled.p`
+  color: white;
+  text-align: left;
+  font-size: 14px;
+  ${fontSize};
+`;
+
+const Title = styled(Text)`
+  font-size: 24px;
+  font-weight: bold;
+  ${fontSize};
```

Comment:
Because `Title` is extending `Text`, you shouldn't need to use a `fontSize` styled-system prop again.